.github/workflows/build.yml

```diff
@@ -12,6 +12,9 @@ env:
   # something like "697/merge", which doesn't work as a version,
   # so wet just set it to master in this case.
   DRONE_BRANCH: ${{ endsWith(github.ref_name, '/merge') && 'master' || github.ref_name }}
+concurrency:
+  group: ${{ github.workflow }}-${{ github.event.pull_request.number || github.ref }}
+  cancel-in-progress: ${{ github.ref != 'refs/heads/master' && !startsWith(github.ref, 'refs/tags/v') }}
```

Comment:
@FrankYang0529 Please help update to skip v1.x branches too (like https://github.com/harvester/harvester/blob/3b9a3e36125de9b4c500703803bd7807e89a7bf5/.github/workflows/build.yml#L15)

Reply:
Updated it. Thanks.